import_vcf cannot import numeric arrays in format field

To report a bug, fill in the information below. 
For support and feature requests, please use the discussion forum: http://discuss.hail.is/

-------------------------------------------------------------------------------------------

### Hail version: devel-abac611 using Apache Spark version 2.2.0

### What you did:
Tried converting joint genotyped Delly vcf file to vdf file.

```
import hail as hl
hl.init(default_reference='GRCh38')
hl.import_vcf('/project/casa/vcf.5k/delly/gcad.sv.delly.5k.vcf.bgz').write('/project/casa/vdf.5k/delly')
```


### What went wrong (all error messages here, including the full java stack trace):

Error reading a GL field. 
##FORMAT=<ID=GL,Number=G,Type=Float,Description="Log10-scaled genotype likelihoods for RR,RA,AA genotypes">

where the entry was:
0/1:-66.2667,0,-25.4754:10000:PASS:5639:13071:8160:2:0:0:13:27

Here are the log and error messages...

```
[Stage 1:======================================================>(415 + 1) / 416]2018-04-12 07:57:52 Hail: INFO: Coerced almost-sorted dataset
[Stage 2:>                                                       (0 + 36) / 416]Traceback (most recent call last):
  File "/restricted/projectnb/genpro/github/hail/delly_vcf2vdf.py", line 3, in <module>
    hl.import_vcf('/project/casa/vcf.5k/delly/gcad.sv.delly.5k.vcf.bgz').write('/project/casa/vdf.5k/delly')
  File "<decorator-gen-546>", line 2, in write
  File "/restricted/projectnb/genpro/github/hail/build/distributions/hail-python.zip/hail/typecheck/check.py", line 481, in _typecheck
  File "/restricted/projectnb/genpro/github/hail/build/distributions/hail-python.zip/hail/matrixtable.py", line 1956, in write
  File "/share/pkg/spark/2.2.0/install/python/lib/py4j-0.10.4-src.zip/py4j/java_gateway.py", line 1133, in __call__
  File "/restricted/projectnb/genpro/github/hail/build/distributions/hail-python.zip/hail/utils/java.py", line 196, in deco
hail.utils.java.FatalError: NumberFormatException: For input string: "-66.2667,0,-25.4754"

Java stack trace:
org.apache.spark.SparkException: Job aborted due to stage failure: Task 48 in stage 2.0 failed 4 times, most recent failure: Lost task 48.3 in stage 2.0 (TID 536, scc-q14.scc.bu.edu, executor 1): is.hail.utils.HailExcput string: "-66.2667,0,-25.4754"
  offending line: chr2  130824417       DEL00068296     AGAACAGGACATCCCAGGCAGCTACAGCCCATC...
        at is.hail.utils.ErrorHandling$class.fatal(ErrorHandling.scala:17)
        at is.hail.utils.package$.fatal(package.scala:26)
        at is.hail.utils.Context.wrapException(Context.scala:23)
        at is.hail.io.vcf.LoadVCF$$anonfun$parseLines$1$$anon$1.hasNext(LoadVCF.scala:741)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at scala.collection.Iterator$$anon$1.hasNext(Iterator.scala:1004)
        at is.hail.rvd.OrderedRVD$$anon$2.hasNext(OrderedRVD.scala:412)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at is.hail.rvd.OrderedRVD$$anonfun$apply$10$$anon$3.hasNext(OrderedRVD.scala:750)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at scala.collection.Iterator$class.foreach(Iterator.scala:893)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1336)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$apply$4.apply(RowStore.scala:774)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$apply$4.apply(RowStore.scala:767)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3.apply(RowStore.scala:767)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3.apply(RowStore.scala:766)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.utils.richUtils.RichHadoopConfiguration$.writeFile$extension(RichHadoopConfiguration.scala:265)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2.apply(RowStore.scala:766)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2.apply(RowStore.scala:763)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1.apply(RowStore.scala:763)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1.apply(RowStore.scala:762)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.utils.richUtils.RichHadoopConfiguration$.writeFile$extension(RichHadoopConfiguration.scala:265)
        at is.hail.io.RichRDDRegionValue$$anonfun$5.apply(RowStore.scala:762)
        at is.hail.io.RichRDDRegionValue$$anonfun$5.apply(RowStore.scala:756)
        at org.apache.spark.rdd.RDD$$anonfun$mapPartitionsWithIndex$1$$anonfun$apply$26.apply(RDD.scala:844)
        at org.apache.spark.rdd.RDD$$anonfun$mapPartitionsWithIndex$1$$anonfun$apply$26.apply(RDD.scala:844)
        at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:38)
        at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:323)
        at org.apache.spark.rdd.RDD.iterator(RDD.scala:287)
        at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:87)
        at org.apache.spark.scheduler.Task.run(Task.scala:108)
        at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:335)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NumberFormatException: For input string: "-66.2667,0,-25.4754"
        at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
        at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
        at java.lang.Double.parseDouble(Double.java:538)
        at scala.collection.immutable.StringLike$class.toDouble(StringLike.scala:284)
        at scala.collection.immutable.StringOps.toDouble(StringOps.scala:29)
        at is.hail.io.vcf.VCFLine.parseDoubleInFormatArray(LoadVCF.scala:371)
        at is.hail.io.vcf.VCFLine.parseAddFormatArrayDouble(LoadVCF.scala:431)
        at is.hail.io.vcf.FormatParser.parseAddField(LoadVCF.scala:483)
        at is.hail.io.vcf.FormatParser.parse(LoadVCF.scala:514)
        at is.hail.io.vcf.LoadVCF$$anonfun$13.apply(LoadVCF.scala:867)
        at is.hail.io.vcf.LoadVCF$$anonfun$13.apply(LoadVCF.scala:848)
        at is.hail.io.vcf.LoadVCF$$anonfun$parseLines$1$$anon$1.hasNext(LoadVCF.scala:717)
        ... 35 more

Driver stacktrace:
        at org.apache.spark.scheduler.DAGScheduler.org$apache$spark$scheduler$DAGScheduler$$failJobAndIndependentStages(DAGScheduler.scala:1499)
        at org.apache.spark.scheduler.DAGScheduler$$anonfun$abortStage$1.apply(DAGScheduler.scala:1487)
        at org.apache.spark.scheduler.DAGScheduler$$anonfun$abortStage$1.apply(DAGScheduler.scala:1486)
        at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)
        at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
        at org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:1486)
        at org.apache.spark.scheduler.DAGScheduler$$anonfun$handleTaskSetFailed$1.apply(DAGScheduler.scala:814)
        at org.apache.spark.scheduler.DAGScheduler$$anonfun$handleTaskSetFailed$1.apply(DAGScheduler.scala:814)
        at scala.Option.foreach(Option.scala:257)
        at org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:814)
        at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:1714)
        at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:1669)
        at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:1658)
        at org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:48)
        at org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:630)
        at org.apache.spark.SparkContext.runJob(SparkContext.scala:2022)
        at org.apache.spark.SparkContext.runJob(SparkContext.scala:2043)
        at org.apache.spark.SparkContext.runJob(SparkContext.scala:2062)
        at org.apache.spark.SparkContext.runJob(SparkContext.scala:2087)
        at org.apache.spark.rdd.RDD$$anonfun$collect$1.apply(RDD.scala:936)
        at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:151)
        at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:112)
        at org.apache.spark.rdd.RDD.withScope(RDD.scala:362)
        at org.apache.spark.rdd.RDD.collect(RDD.scala:935)
        at is.hail.io.RichRDDRegionValue$.writeRowsSplit$extension(RowStore.scala:806)
        at is.hail.rvd.OrderedRVD.writeRowsSplit(OrderedRVD.scala:390)
        at is.hail.variant.MatrixTable.write(MatrixTable.scala:2428)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
        at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
        at py4j.Gateway.invoke(Gateway.java:280)
        at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
        at py4j.commands.CallCommand.execute(CallCommand.java:79)
        at py4j.GatewayConnection.run(GatewayConnection.java:214)
        at java.lang.Thread.run(Thread.java:745)is.hail.utils.HailException: gcad.sv.delly.5k.vcf.bgz: caught java.lang.NumberFormatException: For input string: "-66.2667,0,-25.4754"
  offending line: chr2  130824417       DEL00068296     AGAACAGGACATCCCAGGCAGCTACAGCCCATC...
        at is.hail.utils.ErrorHandling$class.fatal(ErrorHandling.scala:17)
        at is.hail.utils.package$.fatal(package.scala:26)
        at is.hail.utils.Context.wrapException(Context.scala:23)
        at is.hail.io.vcf.LoadVCF$$anonfun$parseLines$1$$anon$1.hasNext(LoadVCF.scala:741)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at scala.collection.Iterator$$anon$1.hasNext(Iterator.scala:1004)
        at is.hail.rvd.OrderedRVD$$anon$2.hasNext(OrderedRVD.scala:412)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at is.hail.rvd.OrderedRVD$$anonfun$apply$10$$anon$3.hasNext(OrderedRVD.scala:750)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at scala.collection.Iterator$class.foreach(Iterator.scala:893)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1336)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$apply$4.apply(RowStore.scala:774)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$apply$4.apply(RowStore.scala:767)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3.apply(RowStore.scala:767)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3.apply(RowStore.scala:766)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.utils.richUtils.RichHadoopConfiguration$.writeFile$extension(RichHadoopConfiguration.scala:265)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2.apply(RowStore.scala:766)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2.apply(RowStore.scala:763)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1.apply(RowStore.scala:763)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1.apply(RowStore.scala:762)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.utils.richUtils.RichHadoopConfiguration$.writeFile$extension(RichHadoopConfiguration.scala:265)
        at is.hail.io.RichRDDRegionValue$$anonfun$5.apply(RowStore.scala:762)
        at is.hail.io.RichRDDRegionValue$$anonfun$5.apply(RowStore.scala:756)
        at org.apache.spark.rdd.RDD$$anonfun$mapPartitionsWithIndex$1$$anonfun$apply$26.apply(RDD.scala:844)
        at org.apache.spark.rdd.RDD$$anonfun$mapPartitionsWithIndex$1$$anonfun$apply$26.apply(RDD.scala:844)
        at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:38)
        at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:323)
        at org.apache.spark.rdd.RDD.iterator(RDD.scala:287)
        at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:87)
        at org.apache.spark.scheduler.Task.run(Task.scala:108)
        at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:335)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)java.lang.NumberFormatException: For input string: "-66.2667,0,-25.4754"
        at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
        at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
        at java.lang.Double.parseDouble(Double.java:538)
        at scala.collection.immutable.StringLike$class.toDouble(StringLike.scala:284)
        at scala.collection.immutable.StringOps.toDouble(StringOps.scala:29)
        at is.hail.io.vcf.VCFLine.parseDoubleInFormatArray(LoadVCF.scala:371)
        at is.hail.io.vcf.VCFLine.parseAddFormatArrayDouble(LoadVCF.scala:431)
        at is.hail.io.vcf.FormatParser.parseAddField(LoadVCF.scala:483)
        at is.hail.io.vcf.FormatParser.parse(LoadVCF.scala:514)
        at is.hail.io.vcf.LoadVCF$$anonfun$13.apply(LoadVCF.scala:867)
        at is.hail.io.vcf.LoadVCF$$anonfun$13.apply(LoadVCF.scala:848)
        at is.hail.io.vcf.LoadVCF$$anonfun$parseLines$1$$anon$1.hasNext(LoadVCF.scala:717)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at scala.collection.Iterator$$anon$1.hasNext(Iterator.scala:1004)
        at is.hail.rvd.OrderedRVD$$anon$2.hasNext(OrderedRVD.scala:412)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at is.hail.rvd.OrderedRVD$$anonfun$apply$10$$anon$3.hasNext(OrderedRVD.scala:750)
        at scala.collection.Iterator$$anon$12.hasNext(Iterator.scala:438)
        at scala.collection.Iterator$class.foreach(Iterator.scala:893)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1336)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$apply$4.apply(RowStore.scala:774)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$apply$4.apply(RowStore.scala:767)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3.apply(RowStore.scala:767)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2$$anonfun$apply$3.apply(RowStore.scala:766)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.utils.richUtils.RichHadoopConfiguration$.writeFile$extension(RichHadoopConfiguration.scala:265)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2.apply(RowStore.scala:766)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1$$anonfun$apply$2.apply(RowStore.scala:763)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1.apply(RowStore.scala:763)
        at is.hail.io.RichRDDRegionValue$$anonfun$5$$anonfun$apply$1.apply(RowStore.scala:762)
        at is.hail.utils.package$.using(package.scala:576)
        at is.hail.utils.richUtils.RichHadoopConfiguration$.writeFile$extension(RichHadoopConfiguration.scala:265)
        at is.hail.io.RichRDDRegionValue$$anonfun$5.apply(RowStore.scala:762)
        at is.hail.io.RichRDDRegionValue$$anonfun$5.apply(RowStore.scala:756)
        at org.apache.spark.rdd.RDD$$anonfun$mapPartitionsWithIndex$1$$anonfun$apply$26.apply(RDD.scala:844)
        at org.apache.spark.rdd.RDD$$anonfun$mapPartitionsWithIndex$1$$anonfun$apply$26.apply(RDD.scala:844)
        at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:38)
        at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:323)
        at org.apache.spark.rdd.RDD.iterator(RDD.scala:287)
        at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:87)
        at org.apache.spark.scheduler.Task.run(Task.scala:108)
        at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:335)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)



Hail version: devel-abac611
Error summary: NumberFormatException: For input string: "-66.2667,0,-25.4754"
```

--- a/python/hail/tests/test_methods.py
+++ b/python/hail/tests/test_methods.py
@@ -1086,6 +1086,10 @@ class Tests(unittest.TestCase):
                               hl.is_defined(t.info.DB) &
                               hl.is_defined(t.info.DS)))
 
+    def test_import_vcf_can_import_float_array_format(self):
+        mt = hl.import_vcf(resource('floating_point_array.vcf'))
+        self.assertTrue(mt.aggregate_entries(hl.agg.all(mt.numeric_array == [1.5, 2.5])))
+
     def test_import_plink(self):
         vcf = hl.split_multi_hts(
             hl.import_vcf(resource('sample2.vcf'),

--- a/src/main/scala/is/hail/io/vcf/LoadVCF.scala
+++ b/src/main/scala/is/hail/io/vcf/LoadVCF.scala
@@ -359,7 +359,7 @@ final class VCFLine(val line: String) {
 
   def parseStringInFormatArray(): String = {
     val start = pos
-    while (!endFormatField())
+    while (!endFormatArrayField())
       pos += 1
     val end = pos
     line.substring(start, end)

--- a/src/test/resources/floating_point_array.vcf
+++ b/src/test/resources/floating_point_array.vcf
@@ -1,0 +1,98 @@
+##fileformat=VCFv4.2
+##hailversion=devel-9441414
+##FORMAT=<ID=numeric_array,Number=.,Type=Float,Description="">
+##contig=<ID=1,length=249250621,assembly=GRCh37>
+##contig=<ID=2,length=243199373,assembly=GRCh37>
+##contig=<ID=3,length=198022430,assembly=GRCh37>
+##contig=<ID=4,length=191154276,assembly=GRCh37>
+##contig=<ID=5,length=180915260,assembly=GRCh37>
+##contig=<ID=6,length=171115067,assembly=GRCh37>
+##contig=<ID=7,length=159138663,assembly=GRCh37>
+##contig=<ID=8,length=146364022,assembly=GRCh37>
+##contig=<ID=9,length=141213431,assembly=GRCh37>
+##contig=<ID=10,length=135534747,assembly=GRCh37>
+##contig=<ID=11,length=135006516,assembly=GRCh37>
+##contig=<ID=12,length=133851895,assembly=GRCh37>
+##contig=<ID=13,length=115169878,assembly=GRCh37>
+##contig=<ID=14,length=107349540,assembly=GRCh37>
+##contig=<ID=15,length=102531392,assembly=GRCh37>
+##contig=<ID=16,length=90354753,assembly=GRCh37>
+##contig=<ID=17,length=81195210,assembly=GRCh37>
+##contig=<ID=18,length=78077248,assembly=GRCh37>
+##contig=<ID=19,length=59128983,assembly=GRCh37>
+##contig=<ID=20,length=63025520,assembly=GRCh37>
+##contig=<ID=21,length=48129895,assembly=GRCh37>
+##contig=<ID=22,length=51304566,assembly=GRCh37>
+##contig=<ID=X,length=155270560,assembly=GRCh37>
+##contig=<ID=Y,length=59373566,assembly=GRCh37>
+##contig=<ID=MT,length=16569,assembly=GRCh37>
+##contig=<ID=GL000207.1,length=4262,assembly=GRCh37>
+##contig=<ID=GL000226.1,length=15008,assembly=GRCh37>
+##contig=<ID=GL000229.1,length=19913,assembly=GRCh37>
+##contig=<ID=GL000231.1,length=27386,assembly=GRCh37>
+##contig=<ID=GL000210.1,length=27682,assembly=GRCh37>
+##contig=<ID=GL000239.1,length=33824,assembly=GRCh37>
+##contig=<ID=GL000235.1,length=34474,assembly=GRCh37>
+##contig=<ID=GL000201.1,length=36148,assembly=GRCh37>
+##contig=<ID=GL000247.1,length=36422,assembly=GRCh37>
+##contig=<ID=GL000245.1,length=36651,assembly=GRCh37>
+##contig=<ID=GL000197.1,length=37175,assembly=GRCh37>
+##contig=<ID=GL000203.1,length=37498,assembly=GRCh37>
+##contig=<ID=GL000246.1,length=38154,assembly=GRCh37>
+##contig=<ID=GL000249.1,length=38502,assembly=GRCh37>
+##contig=<ID=GL000196.1,length=38914,assembly=GRCh37>
+##contig=<ID=GL000248.1,length=39786,assembly=GRCh37>
+##contig=<ID=GL000244.1,length=39929,assembly=GRCh37>
+##contig=<ID=GL000238.1,length=39939,assembly=GRCh37>
+##contig=<ID=GL000202.1,length=40103,assembly=GRCh37>
+##contig=<ID=GL000234.1,length=40531,assembly=GRCh37>
+##contig=<ID=GL000232.1,length=40652,assembly=GRCh37>
+##contig=<ID=GL000206.1,length=41001,assembly=GRCh37>
+##contig=<ID=GL000240.1,length=41933,assembly=GRCh37>
+##contig=<ID=GL000236.1,length=41934,assembly=GRCh37>
+##contig=<ID=GL000241.1,length=42152,assembly=GRCh37>
+##contig=<ID=GL000243.1,length=43341,assembly=GRCh37>
+##contig=<ID=GL000242.1,length=43523,assembly=GRCh37>
+##contig=<ID=GL000230.1,length=43691,assembly=GRCh37>
+##contig=<ID=GL000237.1,length=45867,assembly=GRCh37>
+##contig=<ID=GL000233.1,length=45941,assembly=GRCh37>
+##contig=<ID=GL000204.1,length=81310,assembly=GRCh37>
+##contig=<ID=GL000198.1,length=90085,assembly=GRCh37>
+##contig=<ID=GL000208.1,length=92689,assembly=GRCh37>
+##contig=<ID=GL000191.1,length=106433,assembly=GRCh37>
+##contig=<ID=GL000227.1,length=128374,assembly=GRCh37>
+##contig=<ID=GL000228.1,length=129120,assembly=GRCh37>
+##contig=<ID=GL000214.1,length=137718,assembly=GRCh37>
+##contig=<ID=GL000221.1,length=155397,assembly=GRCh37>
+##contig=<ID=GL000209.1,length=159169,assembly=GRCh37>
+##contig=<ID=GL000218.1,length=161147,assembly=GRCh37>
+##contig=<ID=GL000220.1,length=161802,assembly=GRCh37>
+##contig=<ID=GL000213.1,length=164239,assembly=GRCh37>
+##contig=<ID=GL000211.1,length=166566,assembly=GRCh37>
+##contig=<ID=GL000199.1,length=169874,assembly=GRCh37>
+##contig=<ID=GL000217.1,length=172149,assembly=GRCh37>
+##contig=<ID=GL000216.1,length=172294,assembly=GRCh37>
+##contig=<ID=GL000215.1,length=172545,assembly=GRCh37>
+##contig=<ID=GL000205.1,length=174588,assembly=GRCh37>
+##contig=<ID=GL000219.1,length=179198,assembly=GRCh37>
+##contig=<ID=GL000224.1,length=179693,assembly=GRCh37>
+##contig=<ID=GL000223.1,length=180455,assembly=GRCh37>
+##contig=<ID=GL000195.1,length=182896,assembly=GRCh37>
+##contig=<ID=GL000212.1,length=186858,assembly=GRCh37>
+##contig=<ID=GL000222.1,length=186861,assembly=GRCh37>
+##contig=<ID=GL000200.1,length=187035,assembly=GRCh37>
+##contig=<ID=GL000193.1,length=189789,assembly=GRCh37>
+##contig=<ID=GL000194.1,length=191469,assembly=GRCh37>
+##contig=<ID=GL000225.1,length=211173,assembly=GRCh37>
+##contig=<ID=GL000192.1,length=547496,assembly=GRCh37>
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	Sample0	Sample1	Sample2	Sample3	Sample4	Sample5	Sample6	Sample7	Sample8	Sample9
+1	100000	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100001	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100002	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100003	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100004	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100005	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100006	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100007	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100008	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00
+1	100009	.	A	T	.	.	.	numeric_array	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00	1.50000e+00,2.50000e+00

